itests/org.openhab.binding.avmfritz.tests/itest.bndrun

```diff
@@ -59,6 +59,6 @@ Fragment-Host: org.openhab.binding.avmfritz
 	org.eclipse.jetty.websocket.common;version='[9.4.20,9.4.21)',\
 	org.eclipse.jetty.xml;version='[9.4.20,9.4.21)',\
 	org.ops4j.pax.web.pax-web-api;version='[7.2.11,7.2.12)',\
-	org.openhab.binding.avmfritz;version='[2.5.5,2.5.6)',\
-	org.openhab.binding.avmfritz.tests;version='[2.5.5,2.5.6)',\
-	slf4j.simple;version='[1.7.21,1.7.22)'
+	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
```

Comment:
Related?

Reply:
Difficult to say - it is what the bnd autoresolver makes out of it. Interestingly, the last time it was switched from logback to slf4j, see https://github.com/openhab/openhab-addons/pull/7447/files.

@cweitkamp Did you do that change manually and purposefully at that time?

Reply:
I did not change anything manually. Only use `-DwithResolver` flag. Looking at the history of those files it seem to swap more often. Do you think it can be related to the Java platform?

Reply:
Possibly, yes.
But let's leave it this way - I would want to avoid too many manual changes and rather rely on what the bnd resolver creates as it has to live with that :-) .

Reply:
I can give it a try tomorrow. I have OpenJDK and Oracle Java available. If we prefer `ch.qos.logback.classic` for itests we can put `slf4j.simple` onto the runblacklist.

Reply:
I don't really have any preference. Does it make any difference wrt logs from the test?
Sure, feel free to test, but let us merge this nonetheless already tonight, so that all the other PR builds do not fail anymore, ok?

Reply:
Yes, of course. I do not want to delay this.

AFAIK we do not use any specific features from one or the other implementation. I do not expect any differences for our setup.